storage/src/qmdb/any/ordered/fixed.rs

```diff
@@ -181,59 +183,15 @@ mod test {
                 Operation::Delete(key) => {
                     db.delete(key).await.unwrap();
                 }
-                Operation::CommitFloor(metadata, _) => {
-                    db.commit(metadata).await.unwrap();
+                Operation::CommitFloor(_, _) => {
+                    // CommitFloor consumes self - not supported in this helper.
+                    // Test data from create_test_ops never includes CommitFloor.
+                    panic!("CommitFloor not supported in apply_ops");
                 }
             }
         }
     }
 
-    #[test_traced("WARN")]
```

Comment:
This was meant to be removed earlier -- there's already a version of this test in any/ordered/mod.rs that uses a generic implementation.